staging/src/k8s.io/kubectl/pkg/cmd/create/create_configmap_test.go

```diff
@@ -429,20 +476,28 @@ func TestCreateConfigMap(t *testing.T) {
 	}
 }
 
-func setupEnvFile(lines ...string) func(*testing.T, *ConfigMapOptions) func() {
+func setupEnvFile(filenames []string, lines [][]string) func(*testing.T, *ConfigMapOptions) func() {
```

Comment:
I believe it's needed because after setting up the filenames, we would want to the mapping from `file.env` to `cmenv` over [here](https://github.com/kubernetes/kubernetes/blob/a45aeb626c7f2303c49466ae52833cd410cf88f2/staging/src/k8s.io/kubectl/pkg/cmd/create/create_configmap_test.go#L443)

Reply:
no,  it's no needed 😕